cityofaustin/atd-moped
# inserts and updates moped tags

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/14927
fixes https://github.com/cityofaustin/atd-data-tech/issues/14945

## Testing
**URL to test:** 

local

**Steps to test:**
- restart the hasura cluster or rerun the migrations
- you should see the following records in the `moped_tags` table:
  - Arterial Management - Inspection
  - Arterial Management - Request
  - Street Impact Fee (SIF)
- run the down migrations and you should only see one record with the rest deleted/renamed (previous state)
  - Arterial Management

---
#### Ship list
